lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1273,7 +1280,6 @@ def _process_logging_relation_changed(self, relation: Relation):
 
     def _update_relation_data(self, relation):
         if self._charm.unit.is_leader():
-            relation.data[self._charm.app].update(self._promtail_binary_url)
             logger.debug("Saved promtail binary url: %s", self._promtail_binary_url)
             relation.data[self._charm.app]["endpoints"] = json.dumps(self._endpoints())
             logger.debug("Saved endpoints in relation data")
```

Comment:
I would remove this log entirely, it is very little use in `juju debug-log` for Juju admins.

Reply:
@rbarry82 is taking care of logs in another PR.